tools/ci_build/build.py

```diff
@@ -1656,7 +1658,7 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs):
             if onnx_test:
                 run_subprocess([sys.executable, 'onnxruntime_test_python_backend.py'], cwd=cwd, dll_path=dll_path,
                                python_path=python_path)
-                if not args.disable_contrib_ops:
+                if not args.disable_contrib_ops and not args.use_xnnpack:
```

Comment:
>and not args.use_xnnpack: [](http://example.com/codeflow?start=48&length=25)

why do you disable those tests when xnnpack i used?

Reply:
It counts nodes by op types. XNNPack graph transformer changes operator types. So the old tests wouldn't work.